chapters/en/chapter3/6.mdx

```diff
@@ -1,301 +1,419 @@
+
+The loss curve shows the value of the model's loss over time. Initially, the loss is high and then it gradually decreases, indicating that the model is improving. A decrease in the loss value suggests that the model is making better predictions, as the loss represents the error between the predicted output and the true output.
+
+Now let's shift our focus to the accuracy curve. It represents the model's accuracy over time. The accuracy curve begins at a low value and increases as training progresses. Accuracy measures the proportion of correctly classified instances. So, as the accuracy curve rises, it signifies that the model is making more correct predictions.
+
+One notable difference between the curves is the smoothness and the presence of "plateaus" on the accuracy curve. While the loss decreases smoothly, the plateaus on the accuracy curve indicate discrete jumps in accuracy instead of a continuous increase. This behavior is attributed to how accuracy is measured. The loss can improve if the model's output gets closer to the target, even if the final prediction is still incorrect. Accuracy, however, only improves when the prediction crosses the threshold to be correct.
+
+For example, in a binary classifier distinguishing cats (0) from dogs (1), if the model predicts 0.3 for an image of a dog (true value 1), this is rounded to 0 and is an incorrect classification. If in the next step it predicts 0.4, it's still incorrect. The loss will have decreased because 0.4 is closer to 1 than 0.3, but the accuracy remains unchanged, creating a plateau. The accuracy will only jump up when the model predicts a value greater than 0.5 that gets rounded to 1.
+
+<Tip>
+
+**Characteristics of healthy curves:**
+- **Smooth decline in loss**: Both training and validation loss decrease steadily
+- **Close training/validation performance**: Small gap between training and validation metrics
+- **Convergence**: Curves level off, indicating the model has learned the patterns
+
+</Tip>
+
+### Practical Examples[[practical-examples]]
+
+Let's work through some practical examples of learning curves. First, we will highlight some approaches to monitor the learning curves during training. Below, we will break down the different patterns that can be observed in the learning curves.
+
+#### During Training[[during-training]]
+
+During the training process (after you've hit `trainer.train()`), you can monitor these key indicators:
+
+1. **Loss convergence**: Is the loss still decreasing or has it plateaued?
+2. **Overfitting signs**: Is validation loss starting to increase while training loss decreases?
+3. **Learning rate**: Are the curves too erratic (LR too high) or too flat (LR too low)?
+4. **Stability**: Are there sudden spikes or drops that indicate problems?
+
+#### After Training[[after-training]]
```

Comment:
This structure currently looks like this:

![Screenshot 2025-06-16 at 13 14 04](https://github.com/user-attachments/assets/36b6ed92-4e7e-4e7c-8c96-7687cf6440f1)

Maybe removing 2. and 3. at least :)